alea/submitters/htcondor.py

```diff
@@ -1,12 +1,619 @@
+import subprocess
+import os
+import getpass
+import threading
+import tempfile
+import time
+import tarfile
+import shlex
+import json
 from alea.submitter import Submitter
+import logging
+from pathlib import Path
+from Pegasus.api import *
+from datetime import datetime
+
+DEFAULT_IMAGE = "/cvmfs/singularity.opensciencegrid.org/xenonnt/base-environment:latest"
+WORK_DIR = "/scratch/$USER/workflows"
+TOP_DIR = Path(__file__).resolve().parent
+
+
+# Set up logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger()
 
 
 class SubmitterHTCondor(Submitter):
-    """Submitter for htcondor cluster.
+    """Submitter for htcondor cluster."""
+
+    def __init__(self, *args, **kwargs):
+        # General start
+        super().__init__(*args, **kwargs)
+        self.name = self.__class__.__name__
+        self.htcondor_configurations = kwargs.get("slurm_configurations", {})
```

Comment:
Naive question but why do you get the `slurm_configurations` here? I think I'd have a dedicated `htcondor_configurations` in the runner yaml. This could also have additional fields if you need any!